projects/cn0561/src/cn0561.c

```diff
@@ -115,6 +118,7 @@ int main()
 		.platform_ops = &xil_gpio_ops,
 		.extra = &gpio_extra_param
 	};
+	MAX_SPEED_HZ = 48000000;
```

Comment:
this looks like a re-initialization? is the previous value 24000000 used anywhere? Also for variables use lowercase letters.

Reply:
it should be used when using the cora carrier board and was the default in the original project.

Reply:
ack. but please address the second part of the review (lowercase)